docs/entanglement_forging/how-tos/freeze-orbitals.ipynb

```diff
@@ -201,18 +193,37 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "From here, the problem can be solved following the same steps as in the [tutorials](../tutorials/index.rst)."
+    "Create the ``EntanglementForgingGroundStateSolver``, and specify which orbitals to reduce in the observable."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": 5,
    "metadata": {},
+   "outputs": [],
```

Comment:
Need to tell the EFGSS how to split the observable to match the reduced ansatz via the `orbitals_to_reduce` arg. This is a crucial step and should be included here in this how-to